app/src/main/java/org/xwiki/android/sync/activities/editcontact/EditContactMvpView.java

```diff
@@ -24,16 +24,34 @@
 import okhttp3.ResponseBody;
 import retrofit2.Response;
 
-
+/**
+ * Interface which represent EditContact as {@link MVPView}
```

Comment:
Add few more lines like Its a contract of the Activity, every methods mentioned here first for use.

Reply:
fixed